src/accordion/accordion.ts

```diff
@@ -45,27 +17,46 @@ export class NgbAccordion {
     </div>
   `
 })
-export class NgbAccordionGroup implements OnDestroy {
-  private isDisabled: boolean;
-  private _isOpen: boolean = false;
+export class NgbAccordionGroup {
+  private _isOpen = false;
+  @Input() isDisabled: boolean;
+  @Input() title: string;
+
+  @Input()
+  set isOpen(value: boolean) {
+    this._isOpen = value;
+    if (value) {
+      this.accordion.closeOthers(this);
+    }
+  }
 
-  constructor(private accordion: NgbAccordion) { this.accordion.addGroup(this); }
+  get isOpen(): boolean { return this._isOpen; }
 
-  toggleOpen(event) {
+  constructor(@Inject(forwardRef(() => NgbAccordion)) private accordion: NgbAccordion) {}
+
+  toggleOpen(event): void {
     event.preventDefault();
     if (!this.isDisabled) {
       this.isOpen = !this.isOpen;
     }
   }
+}
 
-  onDestroy(): void { this.accordion.removeGroup(this); }
+@Directive({selector: 'ngb-accordion, [ngb-accordion]', host: {'role': 'tablist', 'aria-multiselectable': 'true'}})
```

Comment:
`'aria-multiselectable': 'true'` shouldn't be hard-coded, I guess you've meant `[attr.aria-multiselectable]: !onlyOneOpen` or sth. BTW, maybe we should rename the var name to `multiselectable`?